backend/src/main/models/Call.ts

```diff
@@ -0,0 +1,31 @@
+'use strict';
+import { Table, Column, Model, PrimaryKey, BelongsTo, ForeignKey, AutoIncrement } from 'sequelize-typescript';
+import { Account } from "./Account";
+
+@Table({ timestamps: false })
+export class Call extends Model {
```

Comment:
According to the ER diagram. Call has a many to many relation with Account. But the way you implements seems like a one-to-many. Many-to-many should have the diamond relationship similar to have events and accounts are linked.

Reply:
I think you got mixed with another relation. Call has a many to one relation with Account.
![image](https://user-images.githubusercontent.com/48056716/135718676-23ec60e9-000e-4204-a6e2-8f5629bc3408.png)

Reply:
Oh I didn't see well. Thanks